### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rohan-sahgal**
Added missing step in instructions to activate virtual environment

### CODE REVIEW

**Review by chriszhuada — APPROVED** [review]
Lgtm

oh fuc, wrong account

File: README.md [hunk 1]
```diff
@@ -25,21 +25,26 @@ SkyAlert is an application that provides real-time weather updates and alerts sp
     python -m venv .venv
     ```
 
-3. Refer to `pyproject.toml` to ensure proper version of Python is installed
+3. Activate the virtual environment:
+    ```shell
+    source .venv/bin/activate
+    ```
+
+4. Refer to `pyproject.toml` to ensure proper version of Python is installed
```
Comment (thereisnoaddress) [inline README.md:33]:
```suggestion
4. Refer to `pyproject.toml` to ensure proper version of Python is installed.
```

File: README.md [hunk 2]
```diff
@@ -25,21 +25,26 @@ SkyAlert is an application that provides real-time weather updates and alerts sp
...
 
-4. Install Poetry to your virtual environment
+5. Install Poetry to your virtual environment
```
Comment (thereisnoaddress) [inline README.md:35]:
```suggestion
5. Install Poetry to your virtual environment:
```

File: README.md [hunk 3]
```diff
@@ -25,21 +25,26 @@ SkyAlert is an application that provides real-time weather updates and alerts sp
...
     ```
     pip install poetry
     ```
 
-5. Run Poetry to install dependencies
+6. Run Poetry to install dependencies
```
Comment (thereisnoaddress) [inline README.md:40]:
```suggestion
6. Run Poetry to install dependencies:
```

File: README.md [hunk 4]
```diff
@@ -25,21 +25,26 @@ SkyAlert is an application that provides real-time weather updates and alerts sp
...
     ```
     poetry install
     ```
 
-6. Navigate into `sky_alert` directory
+7. Navigate into `sky_alert` directory
```
Comment (thereisnoaddress) [inline README.md:45]:
```suggestion
7. Navigate into `sky_alert` directory.
```

File: README.md [hunk 5]
```diff
@@ -25,21 +25,26 @@ SkyAlert is an application that provides real-time weather updates and alerts sp
...
 
-7. Run the application:
+8. Run the application:
```
Comment (thereisnoaddress) [inline README.md:47]:
```suggestion
8. Run the application::
```

**Review by thereisnoaddress — APPROVED** [review]
LGTM, some small nits

**Pull request merged**